news/2023/2023-11-03-interview-with-ephemeral.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Interview with Ephemeral"
+date: 2023-11-03 00:00:00 +0000
+---
+
+Ephemeral, joining the game in August of 2009, has taken an active part in the development of osu!, hitting a large amount of milestones in the game. He has made one of the game's most popular beatmaps, acted as a key role in the development of the Mapping Assistance Team (MAT), and even developed many key features in the game that are still used today! Presently, he has a role of a Community Manager. A few medal hunters decided to organize an interview, which anyone could attend to ask any and all questions their souls had, as well as learn more about this well-accomplished member of this game.
```

Comment:
this intro crucially doesn't mention the reason why eph would be relevant to interview, namely that he makes the medals

Reply:
That was already solved, so yea